CredentialCardDetail.vue

```diff
@@ -75,18 +82,18 @@ export default {
   name: 'CredentialCardDetail',
   components: {CredentialCardField},
   mixins: [credentialMixin],
-  computed: {
-    showDefaultImage() {
```

Comment:
This computed property was removed in the script section but it's not reflected in the html section. Was this your intention @JoshDunnDev ?

Reply:
Where did we move the logic to use the issuer's logo or credential's image?

Reply:
Added logic back, it should not have been removed. I was going to move it to the mixin but decided not to and didn't put it back.